tls.go

```diff
@@ -34,7 +34,7 @@ func setupClientCertificateFromBytes(clientCert, clientKey, caCert []byte, tlsCo
 
 	// Append CA to pool
 	if !tlsConfig.RootCAs.AppendCertsFromPEM(caCert) {
-		return nil, fmt.Errorf("failed to add CA certificate to pool: %s", err)
+		return nil, fmt.Errorf("failed to add CA certificate to pool")
```

Comment:
Ah right, good catch, a remnant from the past when you could actually check for an error from the Append() call.